frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -381,12 +381,7 @@ export default ({
     deleteMessage (message) {
       sbp('gi.actions/chatroom/deleteMessage', {
         contractID: this.currentChatRoomId,
-        data: { id: message.id },
-        hooks: {
-          prepublish: (msg) => {
-            // need to do something
-          }
-        }
+        data: { id: message.id }
```

Comment:
Above this line you forgot to remove the `prepublish` and move the code for `'gi.actions/chatroom/editMessage'`

Reply:
That is done in the PR #1496, because it's for the the Issue #1494.